src/styles/_mixins.scss

```diff
@@ -17,6 +17,8 @@
   background: $white-color;
   padding-bottom: 2em;
   padding-top: 1em;
+  z-index: 1; /* set page z-index below hamburger menu overlay */
+  position: relative;
```

Comment:
Good idea to add these properties to this mixin, but can you create another mixin that this mixin includes, which can also be included in Home root?